docs/product-user-guide/src/main/asciidoc/using-standalone-perspectives-standalone-editor-proc.adoc

```diff
@@ -0,0 +1,23 @@
+[id='using-standalone-perspectives-standalone-editor-proc']
+= Using standalone editor perspective
+
+You can use the standalone editor perspective of {CENTRAL} to access an asset's specific editor. By using this perspective, you can open an asset's editor and can modify the asset as required.
+
+The web link for accessing an asset's standalone editor perspective contains the `standalone` and `path` parameters. The `path` parameter must contain the complete path to the asset and then link must end with `_#StandaloneEditorPerspective_` string. Also, by varying the `path` parameter, you can access a specific asset's editor in the standalone mode.
```

Comment:
"web link" should be "web address" as noted in comments above.

Reply:
Changed as suggested.